docs/AddNewOp.md

```diff
@@ -85,21 +89,31 @@ rank inference at the very least (adding right amount of dimensions to the outpu
 
 [PR 1959](https://github.com/onnx/onnx/pull/1959) is a good example to follow.
 
-### Step 3: PR Review by Operators SIG <a name="step3_new_operator_or_function"></a>
+### Step 3: PR Review by Operators SIG
 
 The [Operators SIG](https://github.com/onnx/sigs/tree/main/operators) is responsible for the operators/functions in the ONNX specification. The SIG regularly meets and reviews PRs.
 
 #### Sign-off
 
 At least two sign-off from the Operators SIG [contributors](https://github.com/onnx/onnx/tree/main/community#community-roles).
 
-### Step 4: ONNX release <a name="step4_new_operator_or_function"></a>
+### Step 4: ONNX release
+
 Once the PR is reviewed and signed off by the Operators SIG, it will be merged. Your new operator/function will be part of the main branch and available to anyone building from source. These are not official releases. ONNX periodically releases official new versions that are a snapshot of the main branch. Your new operator/function will be part of that release.
 
-## Removing operator or function <a name="removing_operator_or_function"></a>
+## Updating an existing operator
+
+The definition of an existing operator may need to be updated when e.g. there are new scenarios or input types to support. The process is largely similar to that for creating a new operator.
+
+### Checklist
+
+Use this checklist when updating an existing operator: https://github.com/onnx/onnx/wiki/Checklist-for-updating-an-existing-operator
```

Comment:
`Add the operator to "onnx/defs/operator_sets.h"` should be `Add the operator to "onnx/defs/operator_sets.h" or "onnx/defs/operator_sets_ml.h"` in the checklist

Reply:
Would be helpful if the checklist mentions the use of auto-update-doc label

Reply:
Done